e2e/pages/events/draftCaseManagementOrderEvent.page.js

```diff
@@ -23,6 +23,15 @@ module.exports = {
     },
   },
 
+  staticFields: {
+    statusRadioGroup: {
+      groupName: '#reviewCaseManagementOrder_cmoStatus',
+      sendToJudge: 'Yes, send this to the judge',
```

Comment:
Ideally if you grouped it within options container e.g.:

```
...
groupName: '#reviewCaseManagementOrder_cmoStatus',
options: {
  sendToJudge: 'Yes, send this to the judge',
  partiesReview: 'No, parties need to review it',
  selfReview: 'No, I need to make changes',
}
...
```

Just a note, it is nice to have - what you have is quite clean already.